sink/structs.go

```diff
@@ -4,5 +4,5 @@ package sink
 type Sink interface {
 	Start() error
 	Stop()
-	Put(data []byte) error
+	Put(key string, data []byte) error
```

Comment:
For consistency's sake, I'd maybe have both `key` and `data` be of type `[]byte`.

Reply:
the IDs in the nomad structs are all string types, so then I would have to convert them all to []bytes.